jerry-core/parser/js/js-parser-expr.c

```diff
@@ -666,23 +666,58 @@ parser_parse_object_literal (parser_context_t *context_p) /**< context */
                                          literal_index,
                                          PARSER_OBJECT_PROPERTY_VALUE);
 
-      lexer_next_token (context_p);
```

Comment:
Why do you remove `lexer_next_token`?

Reply:
I didn't really remove it, I just moved it down into the if statement. If we have the shorthand notation, I cant skip this early because I need to create a literal object from the current token.

Reply:
Currently that's what I'm doing, but I won't emit bytecode if I see what's after the identifier is ill-formed.